packages/api-derive/src/bounties/bounties.ts

```diff
@@ -37,7 +37,7 @@ export function bounties (instanceId: string, api: ApiInterfaceRx): () => Observ
   return memo(instanceId, (): Observable<DeriveBounties> =>
     combineLatest([
       bountyBase.bountyCount<BountyIndex>(),
-      api.query.council.proposalCount<ProposalIndex>()
+      api.query.council ? api.query.council.proposalCount<ProposalIndex>() : of(0)
```

Comment:
I believe line 45 also needs adjustment? (It won't be decorated if no council is present)

Reply:
api.derive.council ? api.derive.council.proposals() : of([])